src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerSearchService.kt

```diff
@@ -45,4 +57,26 @@ class PrisonerSearchService(private val inmateService: InmateService) {
           latestLocationId = it.latestLocationId,
         )
       }
+
+  private fun getInmateDetail(offenderNo: String): InmateDetail {
+    val booking = offenderBookingRepository.findLatestOffenderBookingByNomsId(offenderNo).toNullable()
+    val offender = booking?.offender ?: offenderRepository.findRootOffenderByNomsId(offenderNo).toNullable()
+    if (offender == null) throw EntityNotFoundException.withId(offenderNo)
+    return booking
+      ?.let { offenderTransformer.transform(it) }
+      ?.apply {
+        // TODO These need to be modelled in JPA and set by the OffenderTransformer
+        physicalAttributes = inmateService.getPhysicalAttributes(bookingId)
+        physicalCharacteristics = inmateService.getPhysicalCharacteristics(bookingId)
+        physicalMarks = inmateService.getPhysicalMarks(bookingId)
+        aliases = inmateService.getAliases(bookingId)
+        inmateService.getAllAssessmentsOrdered(bookingId).also {
+          csra = it.firstOrNull { it.isCellSharingAlertFlag }?.classification
+          categoryCode = it.firstOrNull { "CATEGORY" == it.assessmentCode }?.classificationCode
+        }
+      }
```

Comment:
We're still relying on InmateService for the details that aren't modelled by JPA.